src/Network/Network/help/New-AzFirewallPolicyExplicitProxy.md

```diff
@@ -0,0 +1,186 @@
+```
+
+This example creates an explicit proxy with provided settings
+
+## PARAMETERS
+
+### -DefaultProfile
+The credentials, account, tenant, and subscription used for communication with Azure.
+
+```yaml
+Type: IAzureContextContainer
+Parameter Sets: (All)
+Aliases: AzContext, AzureRmContext, AzureCredential
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -EnableExplicitProxy
+Enable Explicit Proxy.
+By default it is disabled.
+
+```yaml
+Type: SwitchParameter
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -EnablePacFile
+Enable PAC File.
+By default it is disabled.
+
+```yaml
+Type: SwitchParameter
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -HttpPort
+Port number for explicit proxy http protocol.
+
+```yaml
+Type: Int32
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -HttpsPort
+Port number for explicit proxy https protocol.
+
+```yaml
+Type: Int32
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -PacFile
+SAS URL for PAC file.
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -PacFilePort
+Port number for firewall to serve PAC file.
+
+```yaml
+Type: Int32
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Confirm
+Prompts you for confirmation before running the cmdlet.
+
+```yaml
+Type: SwitchParameter
+Parameter Sets: (All)
+Aliases: cf
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -WhatIf
+Shows what would happen if the cmdlet runs.
+The cmdlet is not run.
+
+```yaml
+Type: SwitchParameter
+Parameter Sets: (All)
+Aliases: wi
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
+
+## INPUTS
+
+### None
+
+## OUTPUTS
+
+### Microsoft.Azure.Commands.Network.Models.PSAzureFirewallPolicyExplicitProxy
+
+## NOTES
+
+## RELATED LINKS
```

Comment:
Could you add `New-AzFireWallPolicy.md` and `Set-AzFireWallPolicy.md`  to related links? See example: https://github.com/Azure/azure-powershell/blob/00e137537cdcd478bc3ef19d1a26e38aea4aa8ba/src/KeyVault/KeyVault/help/New-AzKeyVault.md?plain=1#L450. Same with `New-AzFireWallPolicy.md` and `Set-AzFireWallPolicy.md` , please add `New-AzFirewallPolicyExplicitProxy.md` to their related link sections.

Reply:
Added related links.

Reply:
please add New-AzFirewallPolicyExplicitProxy.md to the related link sections of New-AzFireWallPolicy.md and Set-AzFireWallPolicy.md , please add New-AzFirewallPolicyExplicitProxy.md as well.

Reply:
Updated related links.